go.mod

```diff
@@ -6,16 +6,13 @@ require (
 	github.com/beevik/etree v1.1.0
 	github.com/bitrise-io/bitrise v0.0.0-20210519130014-380842fb41c1
 	github.com/bitrise-io/envman v0.0.0-20210517135508-b2b4fe89eac5
-	github.com/bitrise-io/go-steputils v0.0.0-20210514150206-5b6261447e77
-	github.com/bitrise-io/go-utils v0.0.0-20210520073355-367fa34178f5
-	github.com/bitrise-io/go-xcode v0.0.0-20210520114025-bdbec53ee59f
+	github.com/bitrise-io/go-steputils v1.0.1
+	github.com/bitrise-io/go-utils v1.0.1
+	github.com/bitrise-io/go-xcode v1.0.3-0.20220222074440-0c316f01fa68
```

Comment:
Will need to be updated, when  https://github.com/bitrise-io/go-xcode/pull/157 merged.